java/src/jmri/jmrix/can/cbus/CbusLightManager.java

```diff
@@ -146,15 +192,27 @@ public Light getBySystemName(@Nonnull String key ) {
     /**
      * {@inheritDoc}
      * 
-     * Forces upper case and trims leading and trailing whitespace.
+     * Forces upper case and trims leading and trailing whitespace, adding +/- if not present.
+     * Does not check for valid prefix, hence doesn't throw NamedBean.BadSystemNameException.
      */
     @CheckReturnValue
     @Override
     public @Nonnull
     String normalizeSystemName(@Nonnull String inputName) {
```

Comment:
For Lights ``normalizeSystemName()`` is our only opportunity to add the special Merg +/- sign to addresses entered as ``int`, since the LightTableTabAction Add pane soon after creates the bean. If rubbish is entered (validation would prevent it, but we do so in tests), the original entry (upper case) is returned.